ddpui/services/dashboard_service.py

```
@@ -30,9 +39,575 @@
+        if dashboard.is_org_default and not force:
+            raise DashboardPermissionError("Cannot delete the organization's default dashboard.")
+
+        # Only allow deletion if the current user is the creator
+        if dashboard.created_by != orguser:
+            raise DashboardPermissionError("You can only delete dashboards you created.")
+
+        # Check if dashboard is landing page for any user
+        if OrgUser.objects.filter(landing_dashboard=dashboard).exists():
+            raise DashboardPermissionError(
+                "Cannot delete a dashboard that is set as landing page for one or more users."
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)